.github/workflows/test_child.yml

```
@@ -205,7 +205,17 @@ jobs:
       with:
         pattern: ctest-log-${{ inputs.job_prefix || inputs.flavor }}-*
         path: ctest-logs
+    - name: Install reporter dependencies
+      run: python3 -m pip install influxdb-client==1.50.0
     - name: Generate combined summary
+      env:
+        INFLUXDB_TEST_REPORT_BUCKET: ${{ secrets.INFLUXDB_TEST_REPORT_BUCKET }}
```

actionlint (1.7.12)

[error] property "influxdb_test_report_bucket" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; ci_telemetry_api_key: string; ci_telemetry_url: string; container_registry: string; github_token: string; hil_pat_token: string}

(expression)